Dockerfile

```
@@ -12,11 +12,9 @@ RUN apt-get update && \
 
 # Don't re-run pip install unless either requirements.txt has changed.
 WORKDIR /single-cell-data-portal
-ADD requirements.txt /single-cell-data-portal/requirements.txt
-ADD backend/corpora/api_server/requirements.txt /single-cell-data-portal/requirements-api.txt
-RUN grep -v requirements.txt requirements.txt > reqs.txt \
-    && cat requirements-api.txt >> reqs.txt \
-    && python3 -m pip install -r reqs.txt
+ADD requirements.txt requirements.txt
```

for consistency across Dockerfiles, perhaps this:
```suggestion
ADD requirements.txt requirements-base.txt
```